wagtail_wordpress_import/test/tests/test_wordpress_item.py

```diff
@@ -327,4 +326,30 @@ def test_items_dict_4(self):
         # self.items_dict[4] = has no wp:post_meta items
         wordpress_item = WordpressItem(self.items_dict[4], self.logger)
         with self.assertRaises(KeyError):
-            post_meta = wordpress_item.clean_wp_post_meta()["wp:postmeta"]
+            wordpress_item.clean_wp_post_meta()["wp:postmeta"]
+
+    def test_items_dict_1_excluded_keys(self):
+        wordpress_item = WordpressItem(self.items_dict[1], self.logger)
+        with self.assertRaises(KeyError):
+            wordpress_item.clean_wp_post_meta()["wp:postmeta"]
```

Comment:
**Suggestion**: We have call `clean_wp_post_meta()` once and store it as e.g. `cleaned_postmeta`, and then look up attributes on that. It would help us to know that the KeyError is from our lookup, and not from within `clean_wp_post_meta()`.